src/components/Home/HomepageMap.tsx

```diff
@@ -67,37 +67,47 @@ const createClusterCustomIcon = function (cluster: MarkerCluster) {
     });
 };
 
+const getStationsMarkers = function (
```

Comment:
Out of curiosity, why did you move it outside the `HomepageMap`?

Reply:
As discussed offline, just moved it outside to improve performace since we don't need to define it every time we render the component